### PR TITLE
Feature/yale 14

### PR DESCRIPTION
**Pull request by afonsobspinto**
closes https://metacell.atlassian.net/browse/YALE-14

- Adds new highlightSearchedInstances prop to the viewers component (set to true for the CPHATE type)
- When admissible (the instance has no user defined color, highlightSearchedInstances is set to true and there are searchTerms) we will apply a grey out to all non-searched instances (we get the match between search term and instance by checking if the instance name includes the search term, case insensitive)

Misc:
- Adds new script to run frontend connected with dev cluster

https://github.com/MetaCell/geppetto-NeuroSCAN/assets/19196034/c1de8443-7ea2-4a3c-a79b-d18825942f5f

### CODE REVIEW

**Comment by afonsobspinto** [comment]
@CodiumAI-Agent /describe

File: applications/neuroscan/frontend/src/components/viewers/viewer.js [hunk 1]
```diff
@@ -165,4 +190,8 @@ class Viewer extends React.Component {
   }
 }
 
-export default withStyles(styles)(Viewer);
+const mapStateToProps = (state) => ({
+  searchTerms: state.search.filters.searchTerms,
```
[the remark this answers is not in the document]
Comment (afonsobspinto) [reply applications/neuroscan/frontend/src/components/viewers/viewer.js:194]: My understanding is that `useSelector` is only available for functional components

[the remark this answers is not in the document]
Comment (zsinnema) [reply applications/neuroscan/frontend/src/components/viewers/viewer.js:194]: @afonsobspinto correct, didnt realize it was not a functional component :-) sorry

[the remark this answers is not in the document]
Comment (afonsobspinto) [reply applications/neuroscan/frontend/src/components/viewers/viewer.js:194]: All good. It was a good point :+1:

**Pull request merged**